packages/server/tests/localAcceptance.env

```diff
@@ -27,6 +27,6 @@ LIMIT_DURATION=90000
 SERVER_REQUEST_TIMEOUT_MS=60000
 MEMWATCH_ENABLED=true
 WRITE_SNAPSHOT_ON_MEMORY_LEAK=false
-HBAR_RATE_LIMIT_TINYBAR=5000000000# 50 HBARs
-HBAR_RATE_LIMIT_DURATION=80000# 80 seconds
-HBAR_RATE_LIMIT_BASIC=4000000000# 40 HBARs
+HBAR_RATE_LIMIT_TINYBAR=12000000000 # 100 HBARs
+HBAR_RATE_LIMIT_DURATION=80000 # 80 seconds
+HBAR_RATE_LIMIT_BASIC=1100000000 # 90 HBARs
```

Comment:
```suggestion
HBAR_RATE_LIMIT_TINYBAR=12000000000 # 120 HBARs
HBAR_RATE_LIMIT_DURATION=80000 # 80 seconds
HBAR_RATE_LIMIT_BASIC=1100000000 # 110 HBARs
```

Reply:
Leftovers from testing, reverted to the values defined in the `main` branch.